pkg/api/v1alpha1/tinkerbellmachineconfig_types.go

```diff
@@ -18,7 +18,7 @@ type TinkerbellMachineConfigSpec struct {
 	// OSImageURL is a URL to the OS image used during provisioning. It must include
 	// the Kubernetes version(s). For example, a URL used for Kubernetes 1.27 could
 	// be http://localhost:8080/ubuntu-2204-1.27.tgz
-	OSImageURL          string               `json:"osImageURL,omitempty"`
```

Comment:
Some versions of bottlerocket are still supported so we shouldn't do this until we stop supporting them fully

Reply:
Yes, they are still supported. But we already made that API change on data center config in this [PR](https://github.com/aws/eks-anywhere/commit/6394dadbefb138af9ffd8d20bc57a6a070d09515) . So this PR is merely trying to make it consistent. And don't think again toggling them back to optional makes sense here given we are eventually deprecating.